From b95c5f4470fc35a8e5806287f8a6418148c57742 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 31 Jul 2016 14:59:34 +0300
Subject: [PATCH 0647/2241] miscdevice: mvebu: add devices for marvell
 telephony drivers

Add 2 miscdevices for Marvell telephony drivers:
- SLIC device - might have different names for different manufacturers
- TAL device - user-space interface for telephony tool

Change-Id: Ib7e2df46196d4a0b6a5b7dcd5d26e7eaa21c99c2
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31564
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
---
 include/linux/miscdevice.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/miscdevice.h b/include/linux/miscdevice.h
index 5430374..cfd39a8 100644
--- a/include/linux/miscdevice.h
+++ b/include/linux/miscdevice.h
@@ -19,6 +19,8 @@
 #define APOLLO_MOUSE_MINOR	7	/* unused */
 #define PC110PAD_MINOR		9	/* unused */
 /*#define ADB_MOUSE_MINOR	10	FIXME OBSOLETE */
+#define SLICDEV_MINOR           73      /* Marvell SLIC control device */
+#define TALDEV_MINOR            74      /* Marvell TAL device */
 #define WATCHDOG_MINOR		130	/* Watchdog timer     */
 #define TEMP_MINOR		131	/* Temperature Sensor */
 #define RTC_MINOR		135
-- 
2.7.4

